hooks/webhook/webhook.go

```diff
@@ -77,8 +80,18 @@ func (i *Invoker) Invoke(request, response interface{}) error {
 		glog.Infof("%s: Will invoke %q", i, reqBodyIndent)
 	}
 
-	// Send request.
 	client := &http.Client{Timeout: i.Timeout}
+	if i.CABundle != "" {
+		caPool := x509.NewCertPool()
```

Comment:
It will be really great if we can add Unit Tests for these changes. Current code lacks UT & your additions will be helpful in this regard.